### DISCUSSION TITLE
How to get alpha mode (a glTF name) from a FBX format?

### DISCUSSION THREAD

**Discussion by SC5Shout** [assimp/assimp#4633] [Q&A] [answered]
I've prepared a textured transparent (alpha channel 0.3f) wall mesh (in Blender, blend mode is Alpha Blend). The glTF format has something like alpha mode: 

```cpp
#define AI_MATKEY_GLTF_ALPHAMODE "$mat.gltf.alphaMode", 0, 0
aiString aiAlphaMode;
material->Get(AI_MATKEY_GLTF_ALPHAMODE, aiAlphaMode);
```

I'd like to support also the FBX format, but I cannot find something similar to that. 
I cannot use AI_MATKEY_OPACITY or AI_MATKEY_COLOR_DIFFUSE (alpha channel), because the values come from a texture that is not loaded yet.

I need the alpha mode to build my material from `aiMaterial`.
I've taken a look into the code but it seems I missed something or there is no such thing.

**Comment by kimkulling** [comment, marked as answer]
The common way to archieve this is:
- Get the right mesh from the scene
- Check which material is referenced in the mesh
- Get the alpha mode for the mesh

If there is no such alpha value accessible in the material there is a bug in the storage for the referenced mesh instance or you just have accesed the wrong material.

Maybe you could provide a small example for the problem if you are not able to fix it on your own in the issues section?
